flink-streaming-java/src/main/java/org/apache/flink/streaming/api/functions/sink/filesystem/PartFileConfig.java

```diff
@@ -34,19 +34,34 @@
 
 	private final String partSuffix;
 
+	/**
+	 * Initiates the {@code PartFileConfig} with default values [{@value DEFAULT_PART_PREFIX}, {@value DEFAULT_PART_SUFFIX}].
+	 */
 	PartFileConfig() {
```

Comment:
The PartFileConfig class itself, and all constructors and methods should be public so people can create it.
I think we should also make the fields non-final and add setters (or a builder).

Ideally the user will create a new PartFileConfig with the defaults and override some parameters.

Reply:
created a builder and make public the constructor